pages/stories/[id].js

```diff
@@ -0,0 +1,21 @@
+import { useRouter } from "next/router";
+import StoryDetailCard from "@/components/StoryDetailCard";
+import stories from "@/public/db";
+
+function StoryDetailPage() {
+  const router = useRouter();
+  const { id } = router.query;
+
+  console.log("stories:", stories);
+  console.log("id:", id);
```

Comment:
`console.log` should be removed as these are not necessary for your feature, but part of your developing process.